ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/search/results/result/card/templates/card.html

```diff
@@ -77,6 +77,14 @@ <h3 class="header">
                             data-asset-share-license="${config.licenseEnabled ? asset.properties['license'] : ''}"
                             data-sly-test="${config.cartEnabled}">${'Add to Cart' @ i18n}</button>
                 </span>
+                <span class="action">
+                    <button class="ui link button"
+			    style="display: none;"
```

Comment:
I think there is a CSS class that will hide; either "hide" or "hidden"?

Reply:
Used hidden class